repl/index.html

```diff
@@ -1,149 +1,8 @@
-          // loaded upon creating the REPL. Each has the following properties:
-          //
-          // src      - The source of the script, a URL
-          // global   - (Optional) a name of a global that the script introduces
-          // exposeAs - (Optional) a name that can alias the global
-          // expose   - (Optional) a list of methods to expose globally
-
-          // Required.
-          // As ramda may be a dependency for other scripts, it will
-          // be intentionally loaded before any other script.
-          ramdaScript: {
-            src    : ramdaUrl,
-            global : 'R'
-          },
-
-          // Optional.
-          // Here we can declare a list of libraries that we wish to have
-          // loaded and exposed in the repl.
-
-          scripts: [
-            {
-              src      : '//cdn.jsdelivr.net/gh/sanctuary-js/sanctuary@latest/dist/bundle.js',
-              global   : 'sanctuary',
-              exposeAs : 'S'
-            },
-            {
-              src    : '//cdn.jsdelivr.net/npm/ramda-fantasy@latest/dist/ramda-fantasy.min.js',
-              global : 'RF',
-              expose : [
-                'Either',
-                'Future',
-                'Identity',
-                'IO',
-                'lift2',
-                'lift3',
-                'Maybe',
-                'Tuple',
-                'Reader'
-              ]
-            }
-          ]
-        });
-
-      }());
-
-    </script>
-
-    <script src="https://code.jquery.com/jquery-1.10.2.min.js"></script>
-    <script src="https://netdna.bootstrapcdn.com/bootstrap/3.0.2/js/bootstrap.min.js"></script>
-
-</body>
-</html>
+<!DOCTYPE html><html class="repl-page"><head><meta charset="UTF-8"><meta content="width=device-width, initial-scale=1" name="viewport"><title>Ramda Documentation</title><link href="../style.css" rel="stylesheet" type="text/css"><link href="https://cdn.rawgit.com/ramda/repl/1.2.0/dist/bundle.css" rel="stylesheet"><script src="https://cdnjs.cloudflare.com/ajax/libs/URI.js/1.18.1/URI.min.js"></script><script src="https://cdn.rawgit.com/ramda/repl/1.2.0/dist/bundle.js"></script><script src="index.js"></script></head><body><input id="open-nav" type="checkbox"><header class="navbar navbar-fixed-top navbar-inverse container-fluid"><div class="container-fluid"><div class="navbar-header"><label class="open-nav" for="open-nav"></label><a class="navbar-brand" href="#"><strong>Ramda</strong><span class="version"> v0.27.1</span></a></div><ul class="nav navbar-nav navbar-left"><li><a href="../">Home</a></li><li><a href="../docs/">Documentation</a></li><li class="active"><a href="../repl/">Try Ramda</a></li></ul><ul class="nav navbar-nav navbar-right"><li><a href="https://github.com/ramda/ramda">GitHub</a></li><li><a href="https://gitter.im/ramda/ramda">Discuss</a></li></ul></div></header><div class="ramda-repl-wrapper"><textarea class="ramda-repl-target"></textarea></div><script>window.gitter = {
```

Comment:
Generated from the PUG template as opposed to from the Handlebars template which seems to have been deprecated. Gitter integration is working.